recipes/yt/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "3.2.3" %}
+
+package:
+  name: yt
+  version: {{ version }}
+
+source:
+  fn: yt-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/y/yt/yt-{{ version }}.tar.gz
+  sha256: 4d6ccf345d9fab965335c9faf8708c7eea79366b81d77f0f302808be3e82c0ed
+
+build:
+  entry_points:
+    - yt = yt.utilities.command_line:run_main
+
+requirements:
+  build:
+    - python
+    - numpy x.x
+    - cython
+    - setuptools 
+    - libpython        # [win]
+    - mingw            # [win]
+  run:
+    - python
+    - numpy x.x
+    - h5py
+    - sympy
+    - matplotlib
+    - ipython
+
+test:
+  requires:
+    - nose
+  commands:
+    - yt -h
+  imports:
+    - yt
+
+about:
+  home: http://yt-project.org/
+  license: BSD 3-clause license
+  summary: Analysis and visualization toolkit for volumetric data
+
+extra:
+    recipe-maintainers:
+      - ngoldbaum
+      - xarthisius
```

Comment:
Sure you guys already chatted about this, but just to confirm, @xarthisius. Are you ok being added as a maintainer on the `yt` feedstock in conda-forge?